code/game/gamemodes/colonialmarines/main.dm

```diff
@@ -140,11 +145,18 @@
 			. = 0
 			break
 
+/datum/game_mode/colonialmarines/proc/has_embryo(var/mob/living/carbon/human/H)//this might be a stupid place to put this
```

Comment:
no,make this proc owned by /mob/living/carbon with no arguments(use src instead of H) and move it in the right file